ui_framework/components/local_nav/_local_search.scss

```diff
@@ -44,6 +48,15 @@
     transform: translateY(-50%); /* 1 */
   }
 
+  .kuiLocalSearchAssistedInput__assistanceLink {
+    display: none;
+  }
+
+  .kuiLocalSearchInput:focus + .kuiLocalSearchAssistedInput__assistance .kuiLocalSearchAssistedInput__assistanceLink,
+  .kuiLocalSearchAssistedInput__assistanceLink:active {
+    display: inherit;
+  }
+
```

Comment:
Good point. How about if we change the help to show up on hover instead of focus:

```scss
.kuiLocalSearchInput:hover {
  & ~ .kuiLocalSearchAssistedInput__assistance {
    .kuiLocalSearchAssistedInput__assistanceLink {
      opacity: 1; /* 1 */
    }
  }
}
```